hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/meta/SigTransactionMetadataBuilder.java

```diff
@@ -173,8 +209,8 @@ private boolean designatesAccountRemoval(@NonNull final AccountID id) {
 
     /**
      * Given a successful key lookup, adds its key to the required signers. Given a failed key
-     * lookup, sets this SigTransactionMetadata's status to either the failure reason of the lookup;
-     * or (if it is non-null), the requested failureStatus parameter.
+     * lookup, sets this {@link SigTransactionMetadata}'s status to either the failure reason of the
+     * lookup; or (if it is non-null), the requested failureStatus parameter.
      *
      * @param result key lookup result
      * @param failureStatus failure reason for the lookup
```

Comment:
I think, we should rename this method. First the default behavior and second the exceptional behavior. I.e. this method should be called `addToKeysOrFail()` IMO.

Reply:
Done.